arm/resource-group.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
If command `New-AzResourceGroup -Name $resourceGroup -Location $Location | Out-Null` is sued, why this file exists?

Reply:
Draft, not ready for review